lapack/gonum/dlangt.go

```diff
@@ -0,0 +1,111 @@
+// Copyright ©2020 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package gonum
+
+import (
+	"math"
+
+	"gonum.org/v1/gonum/lapack"
+)
+
+func (impl Implementation) Dlangt(norm lapack.MatrixNorm, n int, dl, d, du []float64) float64 {
```

Comment:
docs?

Reply:
I can't believe I forgot them but I've added them now. PTAL

Reply:
Did you push that? I can't find them.

Reply:
I did not, I stashed them with other changes I'm working on. But now they're here.